config/sidekiq.yml

```diff
@@ -8,3 +8,4 @@
 :queues:
   - default
   - mailer
```

Comment:
I think its worth adding a comment to say this should be removed once the changes are deployed, as per the commit message.